src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -282,6 +282,28 @@ def get_top_5_issues_by_count_for_file(
     )
     project_ids = [p.id for p in projects]
 
+    stackframe_function_name = lambda i: Function(
+        "arrayElement",
+        (Column("exception_frames.function"), i),
+    )
+    multi_if = []
+    for i in range(-STACKFRAME_COUNT, 0):
+        # if, then conditions
+        multi_if.extend(
```

Comment:
So - we're essentially adding the function name intot he multi_if, IF the name is within the passed `function_names` list?

Reply:
putting the function name in the `function_name` column in the output of the snuba request via the `multi_if`